doc/index.rst

```diff
@@ -68,6 +72,9 @@ For ``OFFLINE`` state, run the script this way.
 
   shutdown_dsm.py -f
 
+In both cases, the script defaults to index=1.
```

Comment:
What does -f do? I read the script's help to see that it short for `--full`, which means "OFFLINE". Perhaps if you inserted "To stop the script" before the code block it would be clearer.

Reply:
Hopefully what I did in the above comment helps clarify this.